MediaService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace HNDunit20
+{
+    class MediaService : ServiceInterface
+    {
+        public double SongPrice
+        {
+            get { return 2.5; }
+        }
+
+        public IList<string> Songs
+        {
+            get
+            {
+                return new List<string>
+                {
+                    "Song 1",
+                    "Song 2",
+                    "Song 3"
+                };
+            }
+        }
+
+        public void ListAll()

```

Comment:
There is a few different ways to approach this but we could use a similar .Append approach from the previous jukebox project